src/components/Navbar/styles.scss

```diff
@@ -121,7 +121,6 @@
     }
 
     &--icon {
-      fill: $tw-clr-icn-secondary;
```

Comment:
This line was overriding the prop for some reason, so we moved the default behavior outside of Navbar.